CODEOWNERS

```diff
@@ -70,7 +73,8 @@ scripts/*				@marc-hb @aborisovich
 
 # tools(old 'soft' repo)
 tools/logger/*				@bkokoszx @akloniex
-tools/topology/*			@ranj063
+tools/topology/**			@ranj063
+tools/topology/topology2/**		@jsarha
```

Comment:
Last pattern wins, so a PR that touches `tools/topology/topology2/foo.conf` only will not add @ranj063. Is that intended?

Reply:
Good catch, thanks @marc-hb . Fixed in latest push.